## Leng Jing 冷靖

No.2005, Songhu Road, Shanghai, China | +86-13220929029 | 20210700129@fudan.edu.cn

<div align=left>
<img src="https://github.com/lengjing0519/lengjing0519.github.io/blob/ed9307e17e89a086e89c1936d33cf193f82047c0/personal%20identification%20photo.jpg" width=300 height=300 />
</div>
 
---
### EDUCATION

**Fudan University**, Shanghai, China\
September 2020 – June 2023 (Expected)\
Master of Science in **Bioinformatics**\
**Awards: Excellent Student Scholarship** (2020-2021)

**Fudan University**, Shanghai, China \
September 2016 – June 2020\
Bachelor of Science in **Biological Sciences** \
**Awards: Third-class Merit-based Scholarship for Undergraduate Students** (2017-2018; 2018-2019)

---
### PAPER

**Analysis of functional differentiation of gene families in spatial transcriptome of mouse embryos**\
Currently writing / The first author 

**Spatiotemporal atlas of organogenesis in development of orchid flowers**\
*Nucleic Acids Research* / Minor revision / The second author 

**A recurrent SHANK1 mutation implicated in autism spectrum disorder causes autistic-like core behaviors in mice via downregulation of mGluR1-IP3R1-calcium signaling** \
*Molecular Psychiatry* / Accept / Co-author

---
### RESEARCH EXPERIENCE  

**Fudan University** &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp;  &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; September 2021 - June 2023  
*Researcher* **under Qi Ji**  
**Project: Analysis of functional differentiation of gene families in spatial transcriptome of mouse embryos**  
In this study, we used the single-cell spatial transcriptome technique combined with own designed spatial dynamic programming method and clustering method based on both gene expression and space location. By integrating mouse embryo datasets at four stages with common data set, we are able to find functionally differentiated gene families , such as the HOX gene.  
- Traditional spatial transtriptome data workflow: quality control; normalization; PCA/U-MAP/T-SNE dimensional reduction; own clustering method not yet published; find marker gene; cell type annotation; pseudotime and cell trajectory analysis; GO/KEGG analysis. 
- Use own designed spatial dynamic programming method to identify spatially similar or differentially expressed gene families. 
- Use Phylomcl and MCScanX combined with four stages’ gene expression information to identify temporal differentially expressed gene families.  

**Fudan University** &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp;  &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; September 2020 - March 2022  
*Researcher* **under Qi Ji**  
**Project: Spatiotemporal atlas of organogenesis in development of orchid flowers**    
In this study, We used spatial transcriptome technique combined with own designed clustering method based on both gene expression and space location and pseudotime analysis. Through innovative application of spatial transcription techniques in orchid flowers, We attempted to reconstruct a more convincing model of floral organ development.  
- Traditional spatial transtriptome data workflow: Quality control; Normalization; PCA/U-MAP/T-SNE dimensional reduction; Own clustering method not yet published; Find marker gene; Cell type annotation; GO/KEGG analysis.  
- We focused on genes involved in floral organ development and And observe their expression at different locations in different slices.   
- Use Monocle3 to reconstruct flower development model and raise reasonable explanation by finding marker genes.  




